tools/mkimage/patches/090-macos-arm64-builing-fix.patch

```diff
@@ -1,47 +0,0 @@
-This patch fixes compilation issues on MacOS arm64.
```

Comment:
Please do not remove 090-macos-arm64-builing-fix.patch , it is not yet upstreamed. It has been sent, but not reviewed by u-boot upstream developers

https://lists.denx.de/pipermail/u-boot/2021-December/468875.html

Reply:
The commit was made a few days ago. I thought this patch was upstream, but simply removed it.

Reply:
Rebased with no conflict.

Reply:
Tested without patched 090 on ath79/nand. Simply rebased patch 090, with no conflicts.

Reply:
@zhanhb thank you, new version of your patch works on darwin arm64 host

Reply:
Thanks for letting me know that.